docs/variant-specific/pink.mdx

```diff
@@ -253,7 +256,24 @@ These conventions apply to any variant with a pink (touched by all ranks) suit.
 
 <BubblegumFinesse />
 
-- _Bubblegum Finesses_ can only be used to get pink cards.
+- Whenever a pink card plays in response to a _Bubblegum_ move, it is guaranteed to the team to be a _Finesse_.
```

Comment:
i dont understand this line.
isn't it false because bubblegum bluffs exists?

Reply:
as far as I understood, there is no such thing as bubblegum-bluffing a pink (or omni) card. those are finesses. (hence the reword above.)

similar to Bob playing red 1 into a red clue on Cathy's red 3. it's not a bluff. either Cathy bombs red 3 or Bob bombs slot 2 next round.